vignettes/TET2.Rmd

```diff
@@ -90,11 +90,17 @@ Is it the case that TET2, IDH1, and IDH2 mutations are exclusive?
 
 # always plot your data
 library(ComplexHeatmap)
-mutations <- t(as.matrix(pData(DNAme)[, c("TET2", "IDH")]))
-Heatmap(mutations, col=c("lightgray","darkred"), name="mutant", column_km=4,
-        column_names_gp = gpar(fontsize = 7))
-
+medata <- pData(DNAme)[, c("TET2", "IDH")]
+mutations <- t(as.matrix(medata))
+Heatmap(mutations,
+    col = c("lightgray", "darkred"), name = "mutant", column_km = 4,
+    column_names_gp = gpar(fontsize = 7)
+)
+
+# get any samples that have both genes mutated
+(non_exclusive  <- medata %>% dplyr::filter(TET2 == 1 & IDH == 1))
```

Comment:
This is cool! Much more concise way to draw out the anomaly! Was this possible because you made **mutations** in two steps? How did you reason this out?

Reply:
I think it may have worked on the original `mutations`. I just made a separate one while trying it out

Reply:
Hey James, this was neat. I wonder what happens when you run the subsequent analysis excluding this sample, or does the code already do this? I didn't think it did, but maybe I didn't follow the logic of your code enough to see it.

Reply:
James. Perhaps you can explain how this code differs from the one in the vignette in its principles? I am trying to rationalize it but cannot quite figure it out. Thanks!

Reply:
Abby: from what I can gather he used dplyr filter to pull out the instances where the subject had both IDH and tet2 mutations. The original code also does this but only puts it in as a it's own cluster in the heatmap. this code identifies the specific subject(s). This may be accomplished also with the original code, but that code does not display the subject(s) ID

Reply:
@agodec It doesn't differ by much. One of the paper's results is that the TET2 and IDH mutations are mutually exclusive. All I've done is show that this is not true for one sample by selecting for rows where both TET2 and IDH are mutated.

Reply:
James, after I saw the heatmap Tim provided us, I was quick to notice that it showed a small group of samples (a tiny sliver) harboring both IDH and TET2 mutations. In light of the authors trying to claim that IDH and TET2 mutations are mutually exclusive, I appreciate the fact that you were keen to write code exposing the samples that harbored both IDH and TET2 mutations. Nice work on exposing the truth!